projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item-list/cart-item-list.component.ts

```diff
@@ -65,6 +65,11 @@ export class CartItemListComponent {
    * In case of a `consignmentEntry`, we need to normalize the data from the orderEntry.
    */
   private resolveItems(items: Item[]): void {
+    if (items == null) {
```

Comment:
As mentioned above we usually tend to stay away from `==` in most cases because `===` is better.

This is one of the cases where the double equal makes sense since it will be true for `null` and `undefined`. That said it's not super readable so you could go with `!items` which is pretty much the same thing but it's more frequent in our code base.

Reply:
Corrected.